linux_os/guide/system/software/updating/ensure_fedora_gpgkey_installed/bash/shared.sh

```diff
@@ -25,13 +25,8 @@ if [ "${RPM_GPG_DIR_PERMS}" -le "755" ]
 then
   # If they are safe, try to obtain fingerprints from the key file
   # (to ensure there won't be e.g. CRC error).
-  # Backup IFS value
-  IFS_BKP="$IFS"
-  IFS=$'\n'
-  GPG_OUT=($(gpg --with-fingerprint --with-colons "${REDHAT_RELEASE_KEY}" | grep '"^fpr' | cut -d ":" -f 10))
+  readarray GPG_OUT < <(gpg --with-fingerprint --with-colons "${REDHAT_RELEASE_KEY}" | grep '"^fpr' | cut -d ":" -f 10)
```

Comment:
I know it was there before, but I have tried this command on my machine and I don't get any line that would start with `"`.  The grep regular expression should be just `^fpr`, without `"` I think. See the similar remediations for RHEL and OL.

Reply:
You are right, other remediations don't have the double quote either.